agents/agents-features/agents-features-opentelemetry/src/jvmMain/kotlin/ai/koog/agents/features/opentelemetry/attribute/Attribute.kt

```diff
@@ -0,0 +1,11 @@
+package ai.koog.agents.features.opentelemetry.attribute
+
+internal interface Attribute {
+    val key: String
+    val value: Any
+
+    val verbose: Boolean
+        get() = false
+
+    fun String.concatKey(other: String) = this.plus(".${other}")
```

Comment:
nit: Putting extension functions inside some namespaces (interfaces/objects/classes) is usually not very convenient and idiomatic way (while we don't have proper context parameters) - it's harder to use them. Just a suggestion (feel free to ignore it if you don't agree with it) - let's make it top level in this file?